### PR TITLE
Documentation Revisions 2020

### PR DESCRIPTION
**Pull request by taylorotwell**

### CODE REVIEW

**Comment by Olili-Foods** [comment]
### Broken Links in Documentation

The _official Jetstream documentation_ link (https://jetstream.laravel.com/1.x/introduction.html) on the **Starter Kits** page of the Laravel Documentation is broken.

**Comment by ninjaparade** [comment]
I approve. ✅

**Comment by gauravmak** [comment]
Taylor discussing the updates - https://www.youtube.com/watch?v=mgyo0kfV7Vg&t=9m45s

File: passwords.md [hunk 1]
```diff
@@ -66,13 +66,15 @@ Next, we will define a route will handle the form request from the "forgot passw
...
 
-> {tip} When manually implementing password resets, you are required to define the contents of the views and routes yourself. If you would like scaffolding that includes all necessary authentication and verification logic, check out [Laravel Jetstream](https://jetstream.laravel.com).
+You may wondering how Laravel knows how to retrieve the user record from your application's database when calling the `Password` facade's `sendResetLink` method. The Laravel password broker utilizes your authentication system's "user providers" to retrieve database records. The user provider used by the password broker is configured within the `passwords` configuration array of your `config/auth.php` configuration file. To learn more about writing custom user providers, consult the [authentication documentation](/docs/{{version}}/authentication#adding-custom-user-providers)
```
Comment (SirNovi) [inline passwords.md:75]: Missing a `be` after the 2. word.

**Comment by driesvints** [comment]
Please send in a PR if you spot a typo, thanks.

**Pull request merged**